src/styles/stars.scss

```diff
@@ -0,0 +1,23 @@
+.card__stars {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  @for $i from 1 through 5 {
+    &--star--#{$i} {
+      background: url(../images/star.svg) var(--background-properties);
+      margin-right: 4px;
+      width: 16px;
+      height: 16px;
+    }
+  }
+
+  :nth-child(-n + 4) {
+    background: url(../images/star-active.svg) var(--background-properties);
+  }
```

Comment:
[ x ] all stars--N modifiers work as expected (Highlight first N stars)
<img width="519" alt="image" src="https://github.com/user-attachments/assets/481dcd7f-065f-46ea-89ee-44ff8207d745">

If the block has `stars--3` modifier you should show 3 active stars and so on